internal/services/logic/logic_app_workflow_resource.go

```diff
@@ -800,6 +922,10 @@ func flattenLogicAppWorkflowAccessControlConfigurationPolicy(input *logic.FlowAc
 		result["allowed_caller_ip_address_range"] = flattenLogicAppWorkflowIPAddressRanges(input.AllowedCallerIPAddresses)
 	}
 
+	if input.OpenAuthenticationPolicies != nil && input.OpenAuthenticationPolicies.Policies != nil {
+		result["open_authentication_policy"] = flattenLogicAppWorkflowOpenAuthenticationPolicy(input.OpenAuthenticationPolicies.Policies)
+	}
```

Comment:
this should always have a value set into the state (as should this method)

Reply:
Thanks for the update